share.c

```diff
@@ -752,6 +760,8 @@ static const char msg_usage2[] NON_RES_RODATA = " [/F:space] [/L:locks]"
 		 "  /U         Uninstall a resident instance.\r\n"
 		 "  /S         Show patch status and table sizes.\r\n"
 		 "  /O         Only operate if already resident, do not install.\r\n"
+		 "  /D         Disable a resident instance.\r\n"
+		 "  /E         Enable a resident instance. (Default.)\r\n"
```

Comment:
I think you need to add these to README.md also.

Reply:
Done.